scripts/azure-windows.yml

```diff
@@ -71,6 +115,37 @@ steps:
     # build the unit test executable before running tests.
     cmake --build $env:AGENT_BUILDDIRECTORY\build\tiledb  -j $env:NUMBER_OF_PROCESSORS --target tiledb_unit --config Release -- /verbosity:minimal
 
+    #vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
+    #---azurite related
```

Comment:
I don't think this styling is necessary, the `if ($env:TILEDB_AZURE -eq "ON") {` let's us know that this is azure-specific.

Reply:
Removed.